wire-library/wire-runtime-swift/src/main/swift/wellknowntypes+Custom/Duration+Codable.swift

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2023 Block Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+#if swift(>=5.7)
+
+@available(macOS 13, iOS 16, watchOS 9, tvOS 16, *)
+private extension Swift.Duration {
```

Comment:
nit: put the `private` on the field instead of the extension. It might have to be `fileprivate`, but I noticed that Swift has gotten more lax about accessing private things if they're on the same type, in the same file, even if they're in different extensions.